src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcCommands.cs

```diff
@@ -19,8 +19,9 @@ internal enum DiagnosticsServerCommandSet : byte
 
     internal enum DiagnosticsServerCommandId : byte
     {
-        OK    = 0x00,
-        Error = 0xFF,
+        OK            = 0x00,
+        ResumeRuntime = 0x01,
+        Error         = 0xFF,
```

Comment:
I made this change as part of dotnet/runtime#38967.  `ResumeRuntime` is under the `Process` command set (`0x04`) instead of the DiagnosticsServer command set.

Reply:
I've made same adjustments in this PR (moving ResumeRuntime to the new Process command set).

Reply:
I may need to move the ResumeRuntime command back to where it was prior to the change in https://github.com/dotnet/runtime/pull/38967, or dual implement it, so that this code can be used on previews prior to which preview that change lands.